src/DependencyInjection/Configuration.php

```diff
@@ -214,4 +215,23 @@ private function createSmsConfiguration(ArrayNodeDefinition $root)
                 ->end()
             ->end();
     }
+
+    private function createLocaleCookieConfiguration(ArrayNodeDefinition $root)
+    {
+        $root
+            ->children()
+                ->arrayNode('locale_cookie')
+                    ->info('Cookie settings for locale cookie')
+                    ->isRequired()
+                    ->children()
+                        ->scalarNode('name')->end()
+                        ->scalarNode('domain')->end()
+                        ->integerNode('expire')->end()
```

Comment:
These values are all very closely related to `setcookie` and Symfony's cookie. Especially the `expire` value, but I think all these values would benefit from documentation (and validation, like the other configuration).